### PR TITLE
tag YOUTH missing concept

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [X] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Looks like 1304 YOUTH was missed.

### CODE REVIEW

**Review by AnnikaTjuka — COMMENTED** [review]
I somehow assumed that youth was the time from puberty to your 20s. But it seems to be a longer time span. Would it make sense to add a new concept?
1304 CHILD (YOUNG HUMAN) includes only "child" glosses and the definition states that it's a person below puberty.

File: concepticondata/conceptlists/Epps-2021-843.tsv [hunk 1]
```diff
@@ -841,4 +841,4 @@ Epps-2021-843-839	839	YARD OR COURT	YARD OR COURT 1 // YARD OR COURT 2	736	YARD
 Epps-2021-843-840	840	YEAR	YEAR	1226	YEAR	år	2293	år	2819	c	0	1	1
 Epps-2021-843-841	841	YOKE	YOKE 1 // YOKE 2 // YOKE 3	14	YOKE	børetre	0				0	0	0
 Epps-2021-843-842	842	YOLK	YOLK	1556	YOLK	eggegule		äggula	3	c	0	0	0
-Epps-2021-843-843	843	YOUTH	YOUTH			ungdom		ungdom		c	0	1	1
+Epps-2021-843-843	843	YOUTH	YOUTH	1304	CHILD (YOUNG HUMAN)	ungdom		ungdom		c	0	1	1
```
Comment (LinguList) [inline concepticondata/conceptlists/Epps-2021-843.tsv:844]: The swedish gloss points to an abstract concept, like German Jugend. So mapping is not justified in my opinion.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 2]
```diff
@@ -475,7 +475,7 @@ Greenhill-2015-2525-996	996	uncle	1327	UNCLE
 Greenhill-2015-2525-997	997	nephew	1063	NEPHEW
 Greenhill-2015-2525-998	998	rafter	829	RAFTER
 Greenhill-2015-2525-999	999	batten		
-Greenhill-2015-2525-1000	1000	youth		
+Greenhill-2015-2525-1000	1000	youth	1304	CHILD (YOUNG HUMAN)
```
Comment (LinguList) [inline concepticondata/conceptlists/Greenhill-2015-2525.tsv:478]: Here, I'd wonder if it is refering to "the youth" or "a youth". Can we tell this from the gloss alone?

File: concepticondata/conceptlists/Nurse-1975-1038.tsv [hunk 3]
```diff
@@ -702,7 +702,7 @@ Nurse-1975-1038-689	689	threaten	v.	threaten	(ku)tisha	1192	THREATEN
 Nurse-1975-1038-690	690	hen, fowl	n.	hen, fowl	kuku	265	FOWL
 Nurse-1975-1038-691	691	shadow, shade	n.	shadow, shade	kivuli	1388	SHADE
 Nurse-1975-1038-692	692	virgin (bride), girl	n.	virgin (bride), girl//virgin(bride), girl	mwali		
-Nurse-1975-1038-693	693	youth	n.	youth	kijana		
+Nurse-1975-1038-693	693	youth	n.	youth	kijana	1304	CHILD (YOUNG HUMAN)
```
Comment (LinguList) [inline concepticondata/conceptlists/Nurse-1975-1038.tsv:705]: We'd need to double-checkl if the gloss *kijana* refers to the abstract concept or to the concept refering to a young person.

File: concepticondata/conceptlists/Vergallito-2020-1121.tsv [hunk 4]
```diff
@@ -438,7 +438,7 @@ Vergallito-2020-1121-436	436	gioia	joy	2524	JOY	5	n	0.9666666667	545.0344828	1	5
 Vergallito-2020-1121-437	437	gioiello	jewel	1029	JEWEL	8	n	0.9666666667	623.5172414	1	544.1034483	-0.1076989198	-0.02677638422	0.7	0.12	4.26	0.32	4.81	v	45.93535749	4.81	0.12	2.042	6.472287076
 Vergallito-2020-1121-438	438	gioioso	joyful			7	a	1	526.8	1	563.4615385	-0.6601568649	-0.08105291657	3.95	0.6	0.88	0.58	4.21	v	35.518591	4.21	0.58	2.044	5.898932107
 Vergallito-2020-1121-439	439	giovane	young	1207	YOUNG	7	a	0.9666666667	566.7241379	1	524.9285714	-0.4648435141	-0.2200217587	3.14	0.33	1.95	0.91	4.49	v	38.44731978	4.49	0.33	2.164	5.895693343
-Vergallito-2020-1121-440	440	gioventù	youth			8	n	1	606.5333333	1	496.4230769	-0.1706705624	-0.5387421225	3.31	0.25	1.29	0.77	4.29	v	40.76690212	4.29	0.25	1.982	5.628472262
+Vergallito-2020-1121-440	440	gioventù	youth	1304	CHILD (YOUNG HUMAN)	8	n	1	606.5333333	1	496.4230769	-0.1706705624	-0.5387421225	3.31	0.25	1.29	0.77	4.29	v	40.76690212	4.29	0.25	1.982	5.628472262
```
Comment (LinguList) [inline concepticondata/conceptlists/Vergallito-2020-1121.tsv:441]: This is a decendent of Latin "iuventud", so it is the abstract concept of "youth", not the "young person".

**Review by LinguList — COMMENTED** [review]
We need to discuss -- as I have indicated -- what we understand by "youth", and at least in three cases, I assume it is not referring to "young person".

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 2 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:478]: Hmm.. don't think I can tell here -- it's got 3 attestations, one from an old SIL list, one from a 18th century Russian scholar (https://en.wikipedia.org/wiki/Nicholas_Miklouho-Maclay),  and a sociolinguistic survey. I'll leave it unmapped.

**Comment by SimonGreenhill** [comment]
Hmm. ok, I think my mental lexicon has "youth" meaning child -> (not an) adult, rather than post puberty. Given the heterogeneity in the terms here, let's close this PR.

**Pull request closed without merging**